.changeset/yellow-glasses-refuse.md

```
@@ -0,0 +1,11 @@
+---
+'@mastra/deployer': patch
+---
+
+fix: add /api route to default public routes to allow unauthenticated
+access
+
+The /api route was returning 401 instead of 200 because it was being caught
+by the /api/_ protected pattern. Adding it to the default public routes
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)